### PR TITLE
Feature/daf 4214 add new method channel

### PR DESCRIPTION
**Pull request by HiroshiOshiro**
### Issue
We need to do some process to hide play button when broadcast ended. 
Although when broadcast ended by CP, in better player, (at least in Android), there is no signal to know the end of broadcast.

### Solution
Add new method channel to tell broadcast was ended to better player library.
This method channel will be call from app side when catch websocket for end of broadcast.

### Specification
https://docs.google.com/spreadsheets/d/1iXu4Vz9wYpT6fHE7oK_W4vyHP_-BGTvk7HTcETFJ_0I/edit#gid=328690871&range=209:210

### TICKET
https://dw-ml-nfc.atlassian.net/browse/DAF-4214

### CODE REVIEW

File: android/src/main/kotlin/com/jhomlala/better_player/BetterPlayerPlugin.kt [hunk 1]
```diff
@@ -309,6 +309,9 @@ class BetterPlayerPlugin : FlutterPlugin, ActivityAware, MethodCallHandler {
                 player.pause()
                 result.success(null)
             }
+            BROADCAST_ENDED -> {
+                // TODO: implement
```
Comment (HiroshiOshiro) [inline android/src/main/kotlin/com/jhomlala/better_player/BetterPlayerPlugin.kt:313]: Wil be update in another PR soon.

**Review by mvn-thanhluu-hn — APPROVED** [review]
LGTM.

File: lib/src/video_player/video_player_platform_interface.dart [hunk 2]
```diff
@@ -138,6 +138,11 @@ abstract class VideoPlayerPlatform {
     throw UnimplementedError('getAbsolutePosition() has not been implemented.');
   }
 
+  /// To  process when broadcast ended.
```
Comment (noboru-i) [inline lib/src/video_player/video_player_platform_interface.dart]: "To"の後に2つのwhite spaceがあるので、1つにしておいてください。

**Review by noboru-i — APPROVED** [review]
LGTM.

@HiroshiOshiro 
すごく細かいところでコメントしているので、そこだけ確認ください。

File: lib/src/video_player/video_player_platform_interface.dart (hunk 2 above)
Comment (HiroshiOshiro) [reply lib/src/video_player/video_player_platform_interface.dart]: Fixed
https://github.com/dwango-nfc/betterplayer/pull/35/commits/00e28884559313870fdcd7dfd3030603398db14e
